src/app/routes/accountRecoveryRoute.js

```diff
@@ -73,14 +73,18 @@ module.exports = (app) => {
           const data = JSON.parse(proxyResData.toString('utf8'));
           if (data.responseCode === 'OK') {
             req.session.otpVerifiedFor = req.body;
-            const encrypt = {
+            const _encrypt = {
               key: req.body.request.key
             }
             if (req.body.request.userId) {
-              encrypt['id'] = req.body.request.userId
+              _encrypt['id'] = req.body.request.userId
             }
             var timeInMin = 5;
-            var validator = encriptWithTime(encrypt, timeInMin);
+            var validator = encriptWithTime(_encrypt, timeInMin);
+            const reqType = req.body.request.type;
+            const dataToEncrypt = {};
```

Comment:
shall we be using expirating time as we did for other places ?

Reply:
No.
We are not using time expiry.